config/routes.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+	resources :dishes, only: [:show, :update] do
+		# Nested route for ingredients for a dish
+		resources :ingredients, only: [:index, :create]
+		resources :dish_ingredients, only: [:create]
+	end
 
-  # Defines the root path route ("/")
-  # root "articles#index"
-end
+	resources :chefs, only: [:show] do
+		# Nested route stuff to show things for a chef
+		resources :dishes, only: [:index]
+		resources :ingredients, only: [:index], controller: 'chef_ingredients'
```

Comment:
Nice work with this route for US 3! Note that the line above is not necessary because we don't need to expose a route for chefs/:id/dishes